core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-code/dot-container-code.component.html

```diff
@@ -1,7 +1,9 @@
 <div class="dot-container-code__content-tab-container">
-    <label class="block p-label-input-required field">{{
-        'message.containers.create.content_type_code' | dm
-    }}</label>
+    <div class="field mb-0">
+        <label class="block p-label-input-required">{{
```

Comment:
Where did you get this class `p-label-input-required`?

Reply:
It is getting from the _misc.scss file.